loader/loader.c

```diff
@@ -5511,6 +5566,31 @@ VKAPI_ATTR VkResult VKAPI_CALL terminator_CreateInstance(const VkInstanceCreateI
             continue;
         }
 
+        if (ptr_instance->icd_tramp_list.scanned_list[i].interface_version < 3 &&
+            (
+#ifdef VK_USE_PLATFORM_XLIB_KHR
+                NULL != icd_term->dispatch.CreateXlibSurfaceKHR ||
+#endif  // VK_USE_PLATFORM_XLIB_KHR
+#ifdef VK_USE_PLATFORM_XCB_KHR
+                NULL != icd_term->dispatch.CreateXcbSurfaceKHR ||
+#endif  // VK_USE_PLATFORM_XCB_KHR
+#ifdef VK_USE_PLATFORM_WAYLAND_KHR
+                NULL != icd_term->dispatch.CreateWaylandSurfaceKHR ||
+#endif  // VK_USE_PLATFORM_WAYLAND_KHR
+#ifdef VK_USE_PLATFORM_ANDROID_KHR
+                NULL != icd_term->dispatch.CreateAndroidSurfaceKHR ||
+#endif  // VK_USE_PLATFORM_ANDROID_KHR
+#ifdef VK_USE_PLATFORM_WIN32_KHR
+                NULL != icd_term->dispatch.CreateWin32SurfaceKHR ||
+#endif  // VK_USE_PLATFORM_WIN32_KHR
+                NULL != icd_term->dispatch.DestroySurfaceKHR)) {
+            loader_log(ptr_instance, VULKAN_LOADER_ERROR_BIT, 0,
+                       "terminator_CreateInstance: Driver %s says it supports interface version %u but still exports VkSurfacekHR"
+                       "entrypoints (Policy #LDP_DRIVER_6)",
```

Comment:
Is LDP_DRIVER_6 correct? These aren't exported functions, they are queried through vkGetInstanceProcAddr. Unless I am mistaken about the code...

Reply:
So, in the case of that policy, the loaders don't even need to support the Create commands.  If they are there at all and they support loader/driver interface 3 (or whatever).... So, yes, this was intentional in the code.  Any way you think we should change the policy?  "Expose support" for versus "export", or "expose via vkGetInstanceProcAddr"?

Reply:
I guess I am just caught up on the term "export" as to me it has a specific association with a binary allowing the function to be queried with `dlsym/GetProcAddress`.

Reply:
It now says:

```
"loader_scanned_icd_add: Driver %s says it supports interface version %u but still returns some surface "
"entrypoints (Policy #LDP_DRIVER_6)",
```

Reply:
Looking at this again, I actually wonder if LDP_DRIVER_8 is the one to issue an error about, not 6.

Reply:
Yep.  Mixed up which one I modified.  Fixed this and the other one I modified.